styles/layout.css

```diff
@@ -0,0 +1,33 @@
+.name {
```

Comment:
I would reorganize your code here so that layout focuses on your major tags, like header, footer, section, etc. I would include both all views and a @media only view as well. https://smacss.com/book/type-layout